server/api/v1/coastlines/coastlines.py

```diff
@@ -0,0 +1,30 @@
+from fastapi import APIRouter, Depends
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.future import select
+from sqlalchemy.orm import joinedload
+
+from server import models, schemas
+# TODO Implement Settings to have hardcoded route prefixes for future use
```

Comment:
What were you planning on doing here?

Reply:
The get_coastlines() method requires those modules in order to properly function, that's why I imported them into the separate coastlines.py file, also it's easier to keep track of what's going on.